opensrp-app/src/main/java/org/smartregister/CoreLibrary.java

```diff
@@ -43,6 +51,18 @@ public static CoreLibrary getInstance() {
     private CoreLibrary(Context contextArg, SyncConfiguration syncConfiguration) {
         context = contextArg;
         this.syncConfiguration = syncConfiguration;
+
+        initP2pLibrary();
```

Comment:
Make starting the P2PLibrary configurable,  by default disable it

Reply:
Cool